website/blog/how-to-discuss-api-changes-during-code-review.md

```diff
@@ -0,0 +1,245 @@
+
+- `optic.yml`
+- `.optic/api/specification`
+- `.optic/ignore`
+- `.optic/.gitignore`
+
+Next, you can add the [Optic GitHub Action](https://github.com/opticdev/optic-changelog) by creating a new file in your repository at `.github/workflows/optic-changelog.yml`:
+
+```yaml
+name: optic-changelog
+on: [pull_request]
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: opticdev/optic-changelog@main
+        with:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          OPTIC_API_KEY: ${{secrets.OPTIC_API_KEY}}
+```
+
+If you haven’t created an Optic account yet, go to [app.useoptic.com](https://app.useoptic.com/) and log in using GitHub. Optic will generate an API Key which you can add to [your repository’s secrets](https://docs.github.com/en/actions/reference/encrypted-secrets).
+
+![Getting your Optic API key](https://i.imgur.com/Be6338j.png)
+
+Once you've checked in the relevant Optic files and GitHub action, push the changes to your repository. This Optic Changelog action will leave comments on any pull requests that include changes to your API, allowing your team to quickly review and discuss changes or new functionality without digging into every line of code.
+
+To demonstrate this, create a new branch in your local repository:
+
+```shell
+git checkout -b api-change
+```
+
+This time, you'll simulate a breaking API change by removing the `'phone'` field from each of the Account records in the `src/data.js` file:
+
+```javascript
+...
+    {
+      '_id': '0',
+      'name': {
+        'first': 'Deidre',
+        'last': 'Hayes'
+      },
+      'email': 'deidre.hayes@undefined.me',
+      // 'phone': '+1 (839) 577-3100',
+      'address': '507 Church Avenue, Heil, Wyoming, 1754',
+      'language': 'English'
+    },
+...
+```
+
+Start your API using `api start`, approve the changes in Optic, and finalize them. Optic will update your specification file as needed.
+
+Commit your changes, push the new branch to your GitHub repository, and open a pull request. The Optic Changelog GitHub Action will show you which endpoints have changed.
+
+![Optic’s GitHub Action summarizes your API changes](https://i.imgur.com/BtmwzdW.png)
+
+You can dive in deeper by clicking “Review” to see exactly which API requests were affected.
+
+![Showing changed API endpoints in Optic](https://i.imgur.com/h4xgAgQ.png)
+
+Anytime a pull request is opened, the Optic GitHub Action will compare the new specification to the specification file in the base branch. Then it summarizes the differences in a comment, allowing your teammates, API consumers, or QA team to quickly assess how your changes might affect the API in the real world.
+
+These open comments are also a great place to have discussions with your team, tag users who need to know about the changes, and suggest updates.
```

Comment:
per https://github.com/opticdev/optic-changelog readme:
I think it's also worth calling out this option to help rope in the right people
```
SUBSCRIBERS (optional) - comma-separated value of GitHub usernames to include in the PR comment, which will notify the users of an API change
```